docs/src/reference/the-traversal.asciidoc

```diff
@@ -2302,6 +2302,281 @@ link:++https://tinkerpop.apache.org/javadocs/x.y.z/core/org/apache/tinkerpop/gre
 link:++https://tinkerpop.apache.org/javadocs/x.y.z/core/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.html#mean-org.apache.tinkerpop.gremlin.process.traversal.Scope-++[`mean(Scope)`],
 link:++https://tinkerpop.apache.org/javadocs/x.y.z/core/org/apache/tinkerpop/gremlin/process/traversal/Scope.html++[`Scope`]
 
+
+[[mergeedge-step]]
+=== MergeEdge Step
+
+The `mergeE()` -step is used to add edges and their properties to a graph in a "create
+if not exist" fashion. The `mergeE` step can also be used to find edges matching a given
+pattern. This step was added to Gremlin as part of the TinkerPop 3.6 release. Prior to
```

Comment:
> This step was added to Gremlin as part of the TinkerPop 3.6 release. Prior to....

This feels more like "Upgrade Documentation" to me rather than matching the style of our reference guide.  I've actually written something similar there. Not sure if you want to move this content there to replace mine, as you add a bit more detail, but I don't think it should stay here.